src/main/webapp/app/overview/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { faPaperPlane } from '@fortawesome/free-solid-svg-icons';
+import { faExpand } from '@fortawesome/free-solid-svg-icons';
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import { MatDialog } from '@angular/material/dialog';
+
+@Component({
+    selector: 'jhi-exercise-chat-widget',
+    templateUrl: './exercise-chat-widget.component.html',
+    styleUrls: ['./exercise-chat-widget.component.scss'],
+})
+export class ExerciseChatWidgetComponent {
+    @ViewChild('chatWidget') chatWidget!: ElementRef;
+    @ViewChild('chatBody') chatBody!: ElementRef;
+    messages: string[] = [];
+    irisMessages: string[] = ['Hey! How can I help you?'];
```

Comment:
Should we translate this initial message or is this only something temporary?

Reply:
Yes, this part will be changed in the next PR